controls/button/button-overview.md

```diff
@@ -8,23 +8,23 @@ slug: button-overview
 
 # Overview
 
-**Telerik Button for Xamarin** is a button control, which enhances the functionality of the standard Xamarin Forms Button, by providing means for customizing its look &amp; feel. UI customization is possible through the provided theme or you can add borders,  transparency, text alignment, and backgrounds and images. 
+**Telerik Button for Xamarin** is a button control that enhances the functionality of the standard Xamarin Forms Button by providing means for customizing its look &amp; feel. UI customization is possible through the provided themes. Additionally, you can add borders, transparency, text alignment, backgrounds, and images.
+
+If you are new to the **Telerik Button for Xamarin** control, see the [Getting Started]({%slug button-getting-started%}) guide for more information on how to add the control to your application.
```

Comment:
The Overview articles are more marketing-related and may also attract new users. That's why I suggest adding the link to the Getting Started article in the beginning.

Reply:
I like that idea